### PR TITLE
MF1933029

### PR DESCRIPTION
**Pull request by ting2696**
Collections,142 same
Calendar,2 same
PrintStream,18	same
Calendar,38	change
String,190 same

### CODE REVIEW

**Comment by gaofengning** [comment]
MF1933020 高凤宁 评估已通过

File: MF1933029/java/Collections_sort_List.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ *  @Map: MutableCollection$mutating func sort(by areInIncreasingOrder: (Self.Element, Self.Element) throws -> Bool) rethrows
+*/
+    import java.util.*;
+    public class Collections_sort_List {
+    /**
+     * input: 1
+     * list List<String>$list
+     * output: 0
+     * List<String>$list-List<String>$ArrayList<String>()& &https://www.yiibai.com/java_generics/java_generics_list.html
+     */
```
Comment (idiomaticrefactoring) [inline MF1933029/java/Collections_sort_List.java]: error

File: MF1933029/java/PrintStream_append_char.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    import java.io.*;
+    public class PrintStream_append_char{
+    /**
+     * input: 2
+     * class0 PrintStream$class0
+     * c c='M'
+     * output: 1
+     * class0 PrintStream$class0
+     * PrintStream$class0-PrintStream$PrintStream(OutputStream out) & OutputStream out=System.out & https://www.yiibai.com/java/io/printstream_append.html
+     */
```
Comment (idiomaticrefactoring) [inline MF1933029/java/PrintStream_append_char.java]: OutputStream$out

File: MF1933029/swift/Character_write_TextOutputStream.swift [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+} 
+
+  class Character_write_TextOutputStream{
+    /**
+     * input: 2
+     * class0 class0: Character = "M"
+     * target TextOutputStream$target
+     * output: 1
+     * target TextOutputStream$target
+     * TextOutputStream$target-TextOutputStream_out() & struct TextOutputStream_out: TextOutputStream & https://developer.apple.com/documentation/swift/textoutputstream
```
Comment (idiomaticrefactoring) [inline MF1933029/swift/Character_write_TextOutputStream.swift]: class0="M"

**Review by idiomaticrefactoring — CHANGES_REQUESTED** [review]
good+1

**Pull request closed without merging**